### PR TITLE
Feature/heartbeat p2p

### PR DESCRIPTION
**Pull request by euniceatcouchbase**
adding heartbeat support for peer to peer.

### CODE REVIEW

File: CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java [hunk 1]
```diff
@@ -52,6 +52,7 @@ public Replicator configure(Args args) throws Exception {
         String tlsAuthType = args.get("tls_auth_type");
         Boolean serverVerificationMode = args.get("server_verification_mode");
         Boolean tlsAuthenticator = args.get("tls_authenticator");
+        String heartbeat = args.get("heartbeat");
```
Comment (mgcouchbase) [inline CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java:55]: What about other platforms ?

Comment (mgcouchbase) [reply CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java:55]: also functional test for Message listener and Url Listener

Comment (sridevi-15) [reply CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java:55]: same as above

**Review by sridevi-15 — CHANGES_REQUESTED** [review]

File: CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java (hunk 1 above)
Comment (euniceatcouchbase) [reply CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java:55]: added support for ios and dotnet.

**Review by mgcouchbase — APPROVED** [review]
it looks good to me. Do we need to write the test for it right ?

File: CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java (hunk 1 above)
Comment (euniceatcouchbase) [reply CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java:55]: I believe the configure() function in PeerToPeerRequestHandler is the only place to set p2p replicator config, so it should work for message endpoint and url endpoint listeners.

Comment (mgcouchbase) [reply CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java:55]: Correct, we need to use that param in the python code right

**Review by sridevi-15 — APPROVED** [review]

**Pull request merged**